store/etcd/etcd.go

```diff
@@ -470,6 +567,32 @@ func (s *Etcd) List(directory string) ([]*store.KVPair, error) {
 	return kv, nil
 }
 
+// ListExt lists child nodes of a given directory, etcd native extension.
+func (s *Etcd) ListExt(directory string) (*store.KVPairExt, error) {
```

Comment:
shouldn't a list return a list like [here](https://github.com/docker/libkv/blob/master/store/etcd/etcd.go#L400) instead of single kv pair ?